controllers/manifest_controller.go

```diff
@@ -42,7 +41,7 @@ func SetupWithManager(
 	}
 
 	runnableListener, eventChannel := listener.RegisterListenerComponent(
-		settings.ListenerAddr, strings.ToLower(labels.OperatorName), verifyFunc,
+		settings.ListenerAddr, strings.ToLower(declarative.OperatorName), verifyFunc,
```

Comment:
I think this is a different operator name than the one coming from labels.

Reply:
As you can see [here](https://github.com/kyma-project/lifecycle-manager/pull/413/files#diff-a2c2307a5750317022e9cecac20fba60845972de6f2e6ec6d1f03492a722d018L9) it was set to `module-manager`. And it is now still set to `module-manager` ([here](https://github.com/kyma-project/lifecycle-manager/pull/413/files#diff-7b7b8e548274819827296add582c47d013f039f15b3502e0944dde5a9a0b326aR11))

This label has just usages in the declarative, since they are already included in the v1beta1 pkg. Thus, I moved the duplicates to the declarative pkg to have a better understanding what it is being used for and to avoid confusion. Unfortunately, we cannot use it directly from v1beta1 due to cyclic dependencies.